lib/plugins/aws/deploy/index.js

```diff
@@ -35,7 +35,13 @@ class AwsDeploy {
         .then(this.uploadDeploymentPackage)
         .then(this.deployFunctions)
         .then(this.updateStack)
-        .then(() => this.serverless.cli.log('Deployment successful!')),
+        .then(() => {
+          const msg = this.options.noDeploy ?
+            'Deployment successful!' :
+            'Did not deploy due to --noDeploy';
```

Comment:
We should probably remove the "Deployment successful" message since deployment never really happened.